tools/medenv/med_environ.xml

```diff
@@ -0,0 +1,141 @@
+<tool id="iabiodiv_smartbiodiv_med_environ" name="Adds the environment variables to a dataset" version="0.0.1" profile="22.05">
+    <description>
+        From the input dataset, it will request Copernicus for environment variables.
+    </description>
+    <requirements>
+        <requirement type="package" version="2.1.3">pandas</requirement>
+        <requirement type="package" version="4.9.3">lxml</requirement>
+        <requirement type="package" version="1.0.4">medenv</requirement>
+    </requirements>
+    <creator>
+        <person givenName="Jérémy" familyName="Fix" url="http://www.github.com/jeremyfix/" email="jeremy.fix@centralesupelec.fr"/>
+    </creator>
+    <command>
+python '$__tool_directory__/med_environ.py'
+    --datafile '${input}'
+    --out_file '${out_file}'
+    --lat_key  '${lat_key}'
+    --long_key '${long_key}'
+    --date_key '${date_key}'
+    --depth_key '${depth_key}'
+    --variables '${env_variables}'
+    --tol_spatial_coordinates '${tol_spatial_coordinates}'
+    --cmems_username '${cmems_username}'
```

Comment:
This should not come from the tool form. Tool parameters are stored for reproducibility and are on purpose exposed to the world - FAIR.

Please take those from the ENV variables from the shell. As shown in @annefou tools.

Reply:
@Marie59 Did you have time to look at how to fix the question on the credentials or shall I try to do something similar than the @annefou tool ?  If I deal with it, I would probably need to loop with you @yvanlebras .

Reply:
For now I don't see another way than how Anne did it.